src/Utils/request/api.tsx

```diff
@@ -1640,6 +1642,35 @@ const routes = {
     },
   },
 
+  // Notes Routes
+  notes: {
+    patient: {
+      listThreads: {
+        path: "/api/v1/patient/{patientId}/thread/",
+        method: "GET",
+        TRes: Type<PaginatedResponse<Thread>>(),
+        TQuery: Type<{ encounter: string }>(),
+      },
+      createThread: {
+        path: "/api/v1/patient/{patientId}/thread/",
+        method: "POST",
+        TRes: Type<Thread>(),
+        TBody: Type<{ title: string; encounter: string }>(),
+      },
+      getMessages: {
+        path: "/api/v1/patient/{patientId}/thread/{threadId}/note/",
+        method: "GET",
+        TRes: Type<PaginatedResponse<Message>>(),
+      },
+      postMessage: {
+        path: "/api/v1/patient/{patientId}/thread/{threadId}/note/",
+        method: "POST",
+        TRes: Type<Message>(),
+        TBody: Type<{ message: string }>(),
+      },
+    },
+  },
```

Comment:
since these APIs are part of encounter, wouldn't it be appropriate to nest this inside `encounter` instead?